app/controllers/users_controller.rb

```diff
@@ -150,6 +150,7 @@ def login_user(user)
     session[:original_uri] = nil
     set_current_taxonomies(user, {:session => session})
     TopbarSweeper.expire_cache
+    user.post_successful_login
```

Comment:
This is also called from User.try_to_login right now, so might make sense to remove it from there (otherwise it's called twice for regular login)?